README.md

```diff
@@ -204,3 +204,13 @@ Each label name is generated with [Lodash template](https://lodash.com/docs#temp
 The `releasedLabels` ```['released<%= nextRelease.channel ? ` on @\${nextRelease.channel}` : "" %> from <%= branch.name %>']``` will generate the label:
 
 > released on @next from branch next
+
+#### addReleases
+
+When set this is option will append all additional releases except the github release to the body of the github release.
```

Comment:
I'd rephrase the sentence, now that it can be added to the top or bottom

```suggestion
Add links to other releases to the GitHub release body.
```